pydantic_evals/pydantic_evals/evals.py

```diff
@@ -0,0 +1,302 @@
+from __future__ import annotations as _annotations
+
+import asyncio
+from collections import defaultdict
+from collections.abc import Awaitable
+from contextlib import nullcontext
+from contextvars import ContextVar
+from dataclasses import dataclass, field
+from typing import Any, Callable, Generic
+
+import logfire
```

Comment:
logfire_api?

Reply:
I think that's a good question, but I think we need to decide if we want to require logfire and/or otel. If we don't require either then we need a plan for counting tokens (or need to decide we just won't). If we require otel but not logfire then I guess logfire_api won't be what we want to use. If we require logfire, then we can just use it here.

Reply:
well pydantic_evals/pyproject.toml says logfire_api, that should be an easy change here, maybe we can relax it to otel later.